src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -150,6 +151,7 @@
     <Compile Include="System\AttributeTests.netstandard.cs" />
     <Compile Include="CompareToTests.cs" />
     <Compile Include="DBNullTests.netstandard.cs" />
+    <Compile Include="DBNullTests.InvalidCast.cs" Condition="'$(TargetGroup)' != 'uap'" />
```

Comment:
Should these be conditioned on uap or uapaot? Based on your PR title I thought this was about uapaot but perhaps this only has a uap configuration.